lib/cylc/scheduler.py

```diff
@@ -697,10 +697,18 @@ def configure_suite(self, reconfigure=False):
 
             self.hold_suite_now = False
             self._pool_hold_point = None
+
+            if self.config.cfg['scheduling']['hold after point']:
+                self._pool_hold_point = get_point(
+                    self.config.cfg['scheduling']['hold after point'])
+
             if self.options.hold_point_string:
                 self._pool_hold_point = get_point(
                     self.options.hold_point_string)
 
+            if self._pool_hold_point:
+                print "[info] suite will hold after " + str(self._pool_hold_point)
```

Comment:
Yeah, that's me copying the style from the above "[info] copying across...". Removing from both.